### PR TITLE
Strips Out Media Players

### PR DESCRIPTION
**Pull request by Fox-McCloud**
We'll see how this goes.

This strips out the nasty media system that was implemented long ago---the one that has resulted in a ton of functionless boomboxes all over the place.

This naturally means the removal of the jukebox in the bar---from my general experience, this doesn't seem to be used by many players *at all*, and it causes endless problems for a lot of people in addition to ahelps ("how come I can't hear it", "I have volume turned down and I still hear it", "I turned it off, it's still playing", "why won't it play at all").....given that it's not that hard to boot up your favorite media player and play your *own* music, this also seems redundant.

We have fixed this so many damn times, yet it keeps on re-breaking, often times even less than a week after it's fixed...it's just not worth maintaining, in my opinion,  especially given, again, you can listen to your *own* music.

Also it's causes some problems and issues with cryodorms, as well, causing runtimes that can also hypothetically break all of say code.

Lastly, from a legal standpoint, if we ever switch hosts to some place like the US, one day, we really won't be able to stream much music, to begin with.

:cl: Fox McCloud
rscdel: Removes the jukebox from the bar
/:cl:

### CODE REVIEW

**Comment by Jaywings** [comment]
Like mutany. This had to happen. It was so BAD.

**Comment by Lady-Luck** [comment]
No arguments here.

**Comment by fludd12** [comment]
My poor, poor jukebox... I've got a SS13 playlist with just the bar mix songs, because memories.

But, besides my sepia-glasses fogging up at this, it really needed to happen.

**Comment by FalseIncarnate** [comment]
There is a part of me that wants the jukebox to stay as a monument to the past. Totally non-functional, of course, just change the description to eternally say it has an "Out of Order" sign stuck to the front of the case.

Of course, this would pretty much just make it a fluff decoration for the bar, so it doesn't make a huge difference if it stays or not.

**Comment by Spacemanspark** [comment]
I'd agree with False, make it a pure fluff item that says "out of order".

**Comment by Dinarzad** [comment]
I kinda like False's suggestion.
Just making some busted up juke box where the other one used to be, sorta like the plaque's at Escape an stuff.

Other then that though, no arguments from me, never really used it, just turned on my own jams when I play SS13

File: code/__DEFINES/preferences.dm [hunk 1]
```diff
@@ -3,12 +3,11 @@
 #define SOUND_MIDI			2
 #define SOUND_AMBIENCE		4
 #define SOUND_LOBBY			8
-#define SOUND_STREAMING		16
-#define SOUND_HEARTBEAT		32
-#define SOUND_BUZZ			64
-#define SOUND_INSTRUMENTS	128
+#define SOUND_HEARTBEAT		16
+#define SOUND_BUZZ			32
+#define SOUND_INSTRUMENTS	64
```
Comment (Krausus) [inline code/__DEFINES/preferences.dm]: Changing bit flags like this will screw up preferences. Have to leave 16 unused for now.

**Comment by tigercat2000** [comment]
The webserver will need to be updated if this goes through @Regen1 @melandor0

**Comment by tigercat2000** [comment]
@Fox-McCloud Remove the `paradise/web/` folder also, it's just the webserver portion of the media manager.

**Comment by Krausus** [comment]
Also, you'll want to remove the `hosttracker` element from the skin, since it only exists to contain the media player's client-side HTML.

**Comment by Plotron** [comment]
I agree - the Jukebox is a pretty fluff item, it should remain in bar with an "Out of Order" sign on it.

**Comment by Rumiluntti** [comment]
Yes, anything to kill that fucking jukebox.

**Comment by FlattestGuitar** [comment]
[linky link](http://i.imgur.com/zEwlGbA.png)

There you go.

**Comment by Necaladun** [comment]
Rip in pieces. I think the out of order sign is an excellent memorial.

Of course I will have to make a fonzie character now, emote hitting it, then play midis.

**Pull request merged**